python/dgl/sparse/matmul.py

```diff
@@ -229,56 +229,113 @@ def spspmm(
     )
 
 
-def mm(
-    A1: Union[SparseMatrix, DiagMatrix],
+def matmul(
```

Comment:
Is this equivalent to @? (Do we support @ for batched multiplication)?

Reply:
If so, change the doc string to
`Multiply two dense/sparse/diagonal matrices, equivalent to ``A @ B`` `